testing/testing.go

```diff
@@ -182,3 +182,23 @@ func GetNetworkID() int {
 
 	return params.StatusChainNetworkID
 }
+
+// GetAccount1PKFile returns the filename for Account1 keystore based
+// on the current network
+func GetAccount1PKFile() string {
```

Comment:
I think having an intention as a comment would be more useful for understanding why it's needed.